pathfinder/paizo/campaign_setting/inner_sea_faiths/_inner_sea_faiths.pcc

```diff
@@ -0,0 +1,47 @@
+# CVS $Revision$ $Author$ -- Tue Dec 15 01:48:08 2015 -- #reformated by PCGen PrettyLST v6.06.00
+CAMPAIGN:Inner Sea Faiths
+GAMEMODE:Pathfinder|Pathfinder_PFS
+TYPE:Paizo Publishing.Campaign Setting
+RANK:4
+STATUS:BETA
```

Comment:
ALPHA is the default setting. Once cleared for release this is changed by the reviewer.